### ISSUE TITLE
Axiom binding the properties dcat:servesDataset and dcat:accessService

### ISSUE THREAD

**Issue by riccardoAlbertoni** [w3c/dxwg#1227]
Considering together with @aisaac the following example,  we were wondering if we need an axiom binding the properties `dcat:servesDataset` and `dcat:accessService`.
Are there any cases in which having a distribution  `:myDatasetDistribution` with access service `:mySparqlService`  does not imply the service also serves `:myDataset`? 

```
:myDataset a  dcat:Dataset ;
  dcat:distribution :myDatasetDistribution .

:myDatasetlDistribution a dcat:Distribution ;
  dcat:accessService :mySparqlService .

:mySparqlService a dcat:DataService ;
  dcterms:conformsTo <https://www.w3.org/TR/sparql11-query/>;
  dcat:endpointURL <http://myendpoint.org/sparql> ;
  dcat:servesDataset :myDataset .
```

If there aren't, we might want to model this, for example with an axiom similar to the following

```
InverseObjectProperties(
dcat:servesDataset
ObjectPropertyChain(dcat:distribution dcat:accessService)
) .
```

**Comment by andrea-perego** [comment]
@riccardoAlbertoni , are you interested in reviving this issue?

Noting it indirectly relates to https://github.com/w3c/dxwg/issues/124 and https://github.com/w3c/dxwg/issues/1334

**Comment by riccardoAlbertoni** [comment]
> @riccardoAlbertoni , are you interested in reviving this issue?
> 
> Noting it indirectly relates to #124 and #1334

yes, I am interested, but before deal with this, I think we need to see how we solve issue #1334
